src/odb/src/defin/definReader.cpp

```diff
@@ -2136,9 +2136,12 @@ bool definReader::createBlock(const char* file)
       _logger->warn(utl::ODB, 149, "DEF parser returns an error!");
     }
   }
-
   defrClear();
-
+  /*
+   * dbModule* top_module = _block->getTopModule();
+  top_module->getChildren().reverse();
+  top_module->getInsts().reverse();
+  */
```

Comment:
There is nothing in the PR but a comment??

Reply:
Sorry was forcing a re-run of the tests just to establish a baseline. Was seeing some evidence of instability. (Tests pass for me with the code in the comment uncommented locally but are failing on Jenkins).